package.json

```diff
@@ -114,7 +115,8 @@
         "preact": "^10.5.7",
         "preact-render-to-string": "^5.1.12",
         "react": "^16.14.0",
-        "react-dom": "^16.14.0"
+        "react-dom": "^16.14.0",
+        "typescript": "^4.1.3"
```

Comment:
We add it to `devDependencies` and `peerDependencies` here - does that need to happen or can it be defined once? My `npm`/`yarn` knowledge isn't enough to know how that works.

Reply:
interesting point, I assume we do not technically need is in `peerDependencies`, but I think that enforcing TS version could be a nice thing to do especailly as v3 TS doesnt support JSX.

Reply:
That makes sense then. We're saying that running this locally you need this typescript version, but if you use this package we also expect you have to have this version as well. That way we should be enforcing it between packages too. 

Sounds like a plan to me